lintro/ai/review/progress.py

```
@@ -0,0 +1,185 @@
+        )
+
+    def on_step(self, *, chunk_index: int, step: str) -> None:
+        """Update the bar description for an in-chunk step."""
+        if self._progress is None or self._task_id is None:
+            return
+        self._progress.update(
+            self._task_id,
+            description=(f"Chunk {chunk_index + 1}/{self._total_chunks}: {step}"),
+        )
+
+    def on_chunk_done(self, *, chunk_index: int) -> None:
```

Ruff (0.15.20)

[warning] Unused method argument: `chunk_index`

(ARG002)